configs/ru_text_config.json

```diff
@@ -43,6 +52,14 @@
                 "repeat_character" : null,
                 "normalize_text" : true,
                 "allowed_chars": "абвгдежзийклмнопрстуфхцчшщъыьэюя "},
+    "dataset": {"drop_space_at_borders" : false,
+                "to_lower_case" : true,
+                "collapse_char_series" : false,
+                "drop_substrings" : ["|", "*"],
+                "replace_chars" :  [],
+                "repeat_character" : "2",
+                "normalize_text" : false,
+                "allowed_chars": "абвгдеёжзийклмнопрстуфхцчшщъыьэюя2 "},
```

Comment:
Is it ok to allow "2" symbol after postprocessing?

Reply:
remove 2